stories/clickable_link.fmf

```diff
@@ -0,0 +1,45 @@
+/clickable_test_link
+    story:
+        As a tester I want to link test verifying Jira issue.
+
+    description:
+        User can click on a link in JIRA issue that returns human-readable metadata of test that covers
+        this issue.
+
+    example:
+      - |
+        url-options: ?test_url=https://github.com/teemtee/tmt&test-name=/tests/core/smoke
+
+    link:
+      - https://issues.redhat.com/browse/TT-207
+
+/clickable_plan_link
+    story:
+        As a tester I want to link plan verifying Jira issue.
+
+    description:
+        User can click on a link in JIRA issue that returns human-readable metadata of plan that covers
+        this issue.
+
+    example:
+      - |
+        url-options: ?plan-url=https://github.com/teemtee/tmt&plan-name=/plans/features/basic&type=plan
+
+    link:
+      - https://issues.redhat.com/browse/TT-207
+
+/clickable_test_and_plan_link
+    story:
+        As a tester I want to link test and plan verifying Jira issue.
+
+    description:
+        User can click on a link in JIRA issue that returns human-readable metadata of test and plan that both cover
+        this issue.
```

Comment:
Let's be specific here as well:

    If the issue is covered by a tmt test which should be executed
    under a specific tmt plan, fmf identifiers of both test and
    plan would be provided to select what exactly would be
    executed. This should allow faster test execution as the whole
    tmt plan does not have to be executed.